ui/app/AppLayouts/Communities/views/CommunityTokenView.qml

```diff
@@ -78,6 +78,20 @@ StatusScrollView {
     signal kickRequested(string name, string contactId, string address)
     signal banRequested(string name, string contactId, string address)
 
+    signal startTokenHoldersManagement(int chainId, string address)
+    signal stopTokenHoldersManagement()
+
+    onVisibleChanged: {
```

Comment:
I had to use visible flag also because the component may exist even if we skip to some other community or section.
Component is destroyed only if we press "Back" button.